src/CMakeLists.template.cmake

```diff
@@ -35,6 +35,13 @@ ${MODEL_DIR}/wrapfunctions.cpp
 add_library(${PROJECT_NAME} ${SRC_LIST_LIB})
 add_library(model ALIAS ${PROJECT_NAME})
 
+# This option can be helpful when using the Intel compiler and compilation of
+# wrapfunctions.cpp fails due to insufficient memory.
+option(ENABLE_WRAPFUNCTIONS_OPTIMIZATIONS "Disable compiler optimizations for wrapfunctions.cpp" ON)
+if(NOT ENABLE_WRAPFUNCTIONS_OPTIMIZATIONS)
```

Comment:
default is false or true here?

Reply:
default is at the very right: ENABLE_WRAPFUNCTIONS_OPTIMIZATIONS -> ON

Text is misleading though. Changed name without text. Fixing.

Reply:
ah, sorry!